benchmarks/wide_dataframe_benchmark.py

```diff
@@ -18,11 +18,7 @@ class WideDataframeBenchmark(_benchmark.Benchmark):
     """
 
     name = "wide-dataframe"
-    valid_cases = (
-        ["use_legacy_dataset"],
-        ["true"],
-        ["false"],
-    )
+    valid_cases = (["use_legacy_dataset"], ["false"])
```

Comment:
Maybe add a comment that this parameter is only kept for historical reasons, but is further ignored nowadays?

Reply:
done, thanks!